isis/src/base/objs/ImageOverlapSet/ImageOverlapSet.cpp

```diff
@@ -33,12 +33,12 @@ namespace Isis {
    *                        addition to logging errors.
    * @see automaticRegistration.doc
    */
-  ImageOverlapSet::ImageOverlapSet(bool continueOnError) {
+  ImageOverlapSet::ImageOverlapSet(bool continueOnError, bool threadedCalculate) {
 
     p_continueAfterError = continueOnError;
     p_writtenSoFar = 0;
     p_calculatedSoFar = -1;
-    p_threadedCalculate = false;
+    p_threadedCalculate = threadedCalculate;
```

Comment:
Just out of curiosity, this was originally forced false here then true later down the line (for reasons? 🤷 ), does just leaving at false or true throughout cause errors?

Reply:
I don't believe it will. The control flow in findimageoverlaps doesn't care about the p_threadedCalculate until after being set to true. If this does cause an error somewhere else I am expecting to see it in the jenkins build.